lib/matplotlib/figure.py

```diff
@@ -32,7 +32,7 @@
 
 from matplotlib.cbook import Stack, iterable
 
-from matplotlib import _image
+from matplotlib import image as mimage
```

Comment:
Actually, it might make sense to keep it as `_image` because `__all__` isn't used in this file. Therefore, we probably don't want to further pollute the namespace.